Added scenarios for version 3 (#538)
ansible/ansible-builder: docs/scenario_guides/scenario_copy.rst

.. _copy_usage:

Copying arbitratory files to EE 
===============================

> Ansible Builder version 3 schema provides option to copy files to EE image.

Reviewer: ```suggestion
Ansible Builder version 3 schema provides the option to copy files to an EE image.
```

> See :ref:`version 3 schema <version_3_format>` for more details.

Reviewer: ```suggestion
See  the :ref:`version 3 schema <version_3_format>` for more details.
```

> In the example below, we will take a look at 

Reviewer: ```suggestion
In the example below, we will take a look at copying arbitratory files to an execution environment.
```

> * Copying arbitratory files to execution environment

Reviewer: ```suggestion
```

.. literalinclude:: copy_ee.yml
   :language: yaml

> In this example, `additional_build_files` section allows you to add `rootCA.crt` to build context directory.

Reviewer: ```suggestion
In this example, the `additional_build_files` section allows you to add `rootCA.crt` to the build context directory.
```

> Once this file is copied to build context directory, it can be used in the build process.

Reviewer: ```suggestion
Once this file is copied to the build context directory, it can be used in the build process.
```

> In order to use, we need to copy the file from the build context directory using `COPY` directive specified in

Reviewer: ```suggestion
In order to use, the file,  we need to copy it from the build context directory using  the `COPY` directive specified in
```

> `prepend_base` step of `additional_build_steps` section.

Reviewer: ```suggestion
the `prepend_base` step of `additional_build_steps` section.
```

> Finally, you can perform any action based upon the copied file, like in this example updating

Reviewer: ```suggestion
Finally, you can perform any action based upon the copied file, such as in this example updating
```
